epub33/core/index.html

```diff
@@ -3846,8 +3846,7 @@ <h5>The <code>meta</code> Element</h5>
 								href="#sec-meta-elem"><code>meta</code> element</a>, which uses different attributes and
 							requires text content, replaces this element.</p>
 
-						<p>For more information about the <code>meta</code> element, refer to its definition in
-							[[OPF-201]].</p>
+						<p>For more information about the <code>meta</code> element, refer to its <a href="http://www.idpf.org/epub/20/spec/OPF_2.0.1_draft.htm#Section2.2">definition</a> in [[OPF-201]].</p>
```

Comment:
For this one, since meta is still a very active part of EPUB, should we not pull the definition into this doc?

Reply:
I leave this to @mattgarrish: I "just" added the deep links without any change on the spec.

Reply:
This is the OPF2 meta element (the one with name and content attributes). As far as I recall, we've always linked back to EPUB 2 for the legacy elements, so this seems fine to me.